pkg/cmd/grafana-server/commands/cli.go

```diff
@@ -52,7 +52,11 @@ func ServerCommand(version, commit, enterpriseCommit, buildBranch, buildstamp st
 
 func RunServer(opts ServerOptions) error {
 	if Version || VerboseVersion {
-		fmt.Printf("Version %s (commit: %s, branch: %s)\n", opts.Version, opts.Commit, opts.BuildBranch)
+		if opts.EnterpriseCommit != "NA" && opts.EnterpriseCommit != "" {
```

Comment:
nit: it would be nice to use a constant

Reply:
Yes, I was actually looking for such a constant but couldn't find one yet. Will try to add one that is used throughout the CLI 🙂

Reply:
there is no:
https://github.com/grafana/grafana/blob/40a41113aafae34026fcdb381ce938e12e57650d/pkg/cmd/grafana/main.go#L14-L19

Reply:
Yes and there is no really good place for one. I think the "least bad place" would be in `pkg/cmd/grafana-cli/commands/cli.go` right now. Otherwise we'd end up with some circle